docs/lib/restapi/fragments/js/fetch.md

```diff
@@ -26,17 +26,17 @@ API
 Example with async/await
 
 ```javascript
-async getData() { 
+function getData() { 
   const apiName = 'MyApiName';
   const path = '/path';
   const myInit = { // OPTIONAL
     headers: {}, // OPTIONAL
   };
 
-  return await API.get(apiName, path, myInit);
+  return API.get(apiName, path, myInit);
```

Comment:
@ffxsam What do you think of the suggestion? Part of it is that this is an example of async/await so I don't want to remove the code of the async/await callouts if we want to show an example

```suggestion
  await API.get(apiName, path, myInit);
  return;
```

Reply:
In this suggested change, you're just making a call to `API.get()` and discarding the results, returning nothing. Shouldn't a `getData()` function return the data itself?